Lindhart.Analyser.MissingAwaitWarning/Lindhart.Analyser.MissingAwaitWarning/Lindhart.Analyser.MissingAwaitWarning.Test/TestData.cs

```diff
@@ -1,40 +1,52 @@
 ﻿namespace Lindhart.Analyser.MissingAwaitWarning.Test
 {
-	internal class TestData
-	{
-		/// <summary>
-		/// File for testing diagnosis
-		/// </summary>
-		public const string TestDiagnosis = @"
+    internal class TestData
+    {
+        /// <summary>
+        /// File for testing diagnosis
+        /// </summary>
+        public const string TestDiagnosis = @"
+using System;
 using System.Threading.Tasks;
+
 namespace AsyncAwaitGames
 {
-    // In my real case, that method just returns Task.
     public interface ICallee { Task<int> DoSomethingAsync(); }
 
-    public class Callee: ICallee
+    public class Callee : ICallee
     {
-        public async Task<int> DoSomethingAsync() => await Task.FromResult(0); // Should not give a warning
+        public async Task<int> DoSomethingAsync() => await Task.FromResult(0);
     }
+
     public class Caller
     {
         public void DoCall()
         {
             ICallee xxx = new Callee();
 
-            // In my real case, the method just returns Task,
-            // so there is no type mismatch when assigning a result 
-            // either.
-            xxx.DoSomethingAsync(); // Should give a warning.
-
-            var task = xxx.DoSomethingAsync(); // Should give a warning when strict rule enabled
-            xxx.DoSomethingAsync().Result; // Should not give a warning
-
-            xxx.DoSomethingAsync().ConfigureAwait(false); // Should give a warning
+            // Should not give a warning
```

Comment:
I think you should add something along the following line to the "Should not give a warning" lines:
`Func<Task> test2 = async () => await Task.Delay(100);`

Reply:
Added.